setup.cfg

```diff
@@ -58,4 +58,6 @@ filterwarnings =
     # setuptools 65+
     # TODO(kmaehashi): Remove distutils from cupy_builder to remove this
     ignore:Absolute path '(.+?)' is being replaced with a relative path '(.+?)' for outputs:DeprecationWarning:distutils
+    # CCCL's cuda_parallel and cuda_cooperative modules currently issue this warning
+    ignore:.*cuda..* module is deprecated.*:DeprecationWarning
```

Comment:
Is this one needed in CuPy? (wondering how Python warnings are emitted from our CCCL usage?)

Reply:
My understanding is that when executing pytest at CuPy root, because of the third party submodule layout, we end up also executing Python tests in CCCL:
https://github.com/NVIDIA/cccl/tree/main/python. 

I consider this a workaround for now, until we figure out a better way in CCCL to avoid Python projects vendoring CCCL to hit this issue, cc @shwina for vis

Reply:
Thanks for clarification. I guess this would be better then: https://github.com/cupy/cupy/pull/8899/commits/8a2d241a159fb744162b5e5845bf951627359e51

I'd keep filters minimum to what actually happens during test runs 😄

Reply:
It's awesome!